### PR TITLE
Millestone 1 project setup

### PR DESCRIPTION
**Pull request by FrancoRosa**
Hello TSEs,
In this project I managed to do:

- Set up a project with create-react-app
- Add the Big.js library
- Deploy project on Heroku [here.](https://big-react-calc.herokuapp.com)

Please give me some feedback

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,5 +31,31 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
+  },
```
Comment (jofer86) [inline package.json]: - Here we are missing the prop types module please add it.
image

**Review by jofer86 — CHANGES_REQUESTED** [review]
## `const STATUS = "CHANGES REQUIRED"`⚠️ ⚠️ ⚠️ 
## :wave: :wave: Hello! I'm Jorge, and  I will be conducting this code review. Let me get a closer look 🔎 🔍 🔎 🔍 🕵🏼 
- There are a couple of things that need to improve before this milestone can be approved, first remove all unnecessary boilerplate files that get generated with create-react-app as you are not going to be using any of those files, just left out what you are actually going to use 
image
- See below for the next required change.

**Comment by FrancoRosa** [comment]
Thank you @jofer86
All changes requested were made.

**Review by jofer86 — APPROVED** [review]
## `const PROJECT_STATUS = "APPROVED"` 🚀 🚀 🚀 
## :wave: :wave: Hello! I'm Jorge, and  I will be conducting this code review. Let me get a closer look 🔎 🔍 🔎 🔍 🕵🏼 
- I can test that the changes were done, you may move along!!! congrats, on to the next milestone now!!
--------------------------------------------------------------------
**Rock on**  🤘🏼 
**_Jorge Fernando Rincón Figueroa._** 
Technical Support Engineer.

**Pull request merged**